assets/js/components/Message/status.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+const Status = ({ status, winner }) => {
+    if (status == "winner") {
+      return (
+        <div className="winner">
+          {winner} is the {status}
+        </div>
+      );
+    }
+    if (status == "draw") {
```

Comment:
Couldn't this be a case statement/switch?

Reply:
I thought if/else statements were preferred over switch statements. Is that not the case?

Reply:
It depends on the situation. For this it seems like a switch would be better @Bwoodfork?